### PR TITLE
Add bindings to SuggestionViewCell

### PR DESCRIPTION
**Pull request by willsb**
No more placeholders.

Squash whenever

### CODE REVIEW

File: Toggl.Daneel/Converters/BoolToConstantValueConverter.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Globalization;
+using MvvmCross.Platform.Converters;
+
+namespace Toggl.Daneel.Converters
+{
+    public sealed class BoolToConstantValueConverter : MvxValueConverter<bool, nfloat>
```
Comment (arturs105) [inline Toggl.Daneel/Converters/BoolToConstantValueConverter.cs]: What about making this generic `BoolToConstantValueConverter<T> : MvxValueConverter<bool, T>`? That way we wouldn't need any more bool to something converters and `BoolToUIColorConverter` could also be removed.

File: Toggl.Daneel/ViewSources/SuggestionsTableViewSource.cs [hunk 2]
```diff
@@ -54,5 +69,7 @@ protected override UITableViewCell GetOrCreateCellFor(UITableView tableView, NSI
             => tableView.DequeueReusableCell(item == null ? emptyCellIdentifier : cellIdentifier, indexPath);
 
         public override nfloat GetHeightForRow(UITableView tableView, NSIndexPath indexPath) => 64;
+
+        public override nfloat GetHeightForHeader(UITableView tableView, nint section) => 54;
```
Comment (arturs105) [inline Toggl.Daneel/ViewSources/SuggestionsTableViewSource.cs]: I think it's 45 in Zeplin.

File: Toggl.Daneel/Converters/BoolToConstantValueConverter.cs (hunk 1 above)
Comment (willsb) [inline Toggl.Daneel/Converters/BoolToConstantValueConverter.cs]: Nice idea 👌

File: Toggl.Daneel/ViewSources/SuggestionsTableViewSource.cs (hunk 2 above)
Comment (willsb) [inline Toggl.Daneel/ViewSources/SuggestionsTableViewSource.cs]: Correct

**Pull request merged**